docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/FeatureSelection/SelectFeaturesBasedOnCountMultiColumn.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class SelectFeaturesBasedOnCountMultiColumn
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            var rawData = GetData();
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
+
+            Console.WriteLine("Contents of column 'GroupB'");
+            PrintDataColumn(data, "GroupB");
+            // 4       NaN     6
+            // 4       5       6
+            // 4       5       6
+            // 4       NaN     NaN
+
+            Console.WriteLine("Contents of column 'GroupC'");
+            PrintDataColumn(data, "GroupC");
+            // NaN     8       9
+            // NaN     8       9
+            // NaN     8       9
+            // 7       8       9
+
+            // Second, we define the transformations that we apply on the data. Remember that an Estimator does not transform data
+            // directly, but it needs to be trained on data using .Fit(), and it will output a Transformer, which can transform data.
+
+            // We will use the SelectFeaturesBasedOnCount transform estimator, to retain only those slots which have 
+            // at least 'count' non-default values per slot.
+
+            // Multi column example : This pipeline uses two columns for transformation
```

Comment:
>This pipeline uses two columns for transformation [](start = 38, length = 49)

i think this is clearer: this pipeline transform two columns using the same options.

just want to make sure it's clear that columns are transformed independently and are not mixed